src/openforms/registrations/contrib/objects_api/models.py

```diff
@@ -1,11 +1,19 @@
+from functools import partial
+
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
 from solo.models import SingletonModel
 from zgw_consumers.constants import APITypes
 
+from openforms.registrations.contrib.email.models import render_with_language
```

Comment:
Pulling this out of `contrib.email` feels  off. Is setting the language critical? Also see my remark on the template itself. I'm not sure i18n is needed during render. But I might be wrong.